_data/users.yml

```
@@ -281,6 +281,14 @@
     wide array of workloads ranging from ad-hoc to batch queries.
   urltext:
   url: https://www.myntra.com/
+- name: Naver
+  anchor: naver
+  logo: /assets/images/logos/naver.png
+  logosmall: /assets/images/logos/naver-small.png
+  testimonial: |
+    TBD
```

Am not sure about the width of the sentences but here is our testimonial :) 
```suggestion
    Naver Corporation is a global ICT company, providing South Korea’s number one 
    search portal "NAVER". Naver uses Trino for data analytics and business intelligence
    (BI) purposes. Trino allows Naver to query and process massive amounts of data from 
    different sources, such as user logs and clickstream data, in real-time. This enables Naver 
    to deliver more relevant and accurate search results to users based on their preferences.
```

---

Could it be possible to hold this pr from merging for Naver Corp? Sry for inconvenience

---

Here is the updated testimonial :) 

```suggestion
Naver Corporation is a global ICT company, providing South Korea’s number one search portal "NAVER". Naver uses Trino for data analytics and business intelligence (BI) purposes. Trino provides data scientists with quick access to their data lake, via a standard SQL interface. Trino process massive amounts of data from different sources, such as user logs and clickstream data, in real-time.
```